main.py

```diff
@@ -1,16 +1,19 @@
+import os
 from fastapi import FastAPI
 import uvicorn
 from endpoints import Endpoints
 
-
-GITHUB_PLUGINS_JSON_URL = "https://raw.githubusercontent.com/cheshire-cat-ai/awesome-plugins/main/plugins.json"
-DEFAULT_PAGE_SIZE = 10
-CACHE_DURATION_MINUTES = 1440  # Set cache duration (1 day in minutes)
+GITHUB_PLUGINS_JSON_URL = os.environ["GITHUB_PLUGINS_JSON_URL"]
```

Comment:
we should use `os.getenv("NAME", "default_value")` so you can give a default also when docker is not used